packages/devtools_app/lib/src/inspector/flutter/inspector_tree_flutter.dart

```diff
@@ -83,18 +84,21 @@ class _InspectorTreeRowState extends State<_InspectorTreeRowWidget>
 
 class InspectorTreeControllerFlutter extends Object
     with InspectorTreeController, InspectorTreeFixedRowHeightController {
+  // Controller for controlling custom animation in the Tree
+  AnimationController animationController;
+
   /// Client the controller notifies to trigger changes to the UI.
-  InspectorControllerClient get client => _client;
-  InspectorControllerClient _client;
-  set client(InspectorControllerClient value) {
-    if (_client == value) return;
-    // Do not set a new client if there is still an old client.
-    assert(value == null || _client == null);
-    _client = value;
-
-    if (config.onClientActiveChange != null) {
-      config.onClientActiveChange(value != null);
-    }
+  List<InspectorControllerClient> _clients = [];
```

Comment:
lets support clients on the InspectorController not the InspectorTreeController.  The api for this Client class is a bit too coupled to implementing the inspector tree.

Reply:
done on the latest revision but unsure whether the solution is correct or not